server/routes/index.js

```diff
@@ -37,7 +37,6 @@ function getRouter(appType) {
                 csrfToken: req.csrfToken(),
                 ivisConfig: JSON.stringify(ivisConfig),
                 indexFile: appType === AppType.TRUSTED ? getTrustedUrl('client/index-trusted.js') : getSandboxUrl('client/index-sandbox.js'),
-                bodyClass: appType === AppType.TRUSTED ? "trusted" : "sandbox",
```

Comment:
Tohle si myslím, že chceme zachovat. "trusted" sice možná žádné styly nemá, ale "sandbox" určitě ano (viz řádek 26 v `ivis.scss`)